doc/rst/logging.rst

```diff
@@ -52,14 +54,10 @@ DEBUG_PRINT_DISABLE_[debug unit]
   debug unit. If set to 1, this will override the default set by
   ``DEBUG_PRINT_ENABLE``.
 
-|newpage|
-
-|appendix|
-
 Known Issues
-------------
+============
 
 There are no known issues with this library.
 
 
-.. include:: ../../../CHANGELOG.rst
```

Comment:
I think to make this not break due to different title levels you need to have 

************
Changelog 
***********

Or similar

Reply:
Ah ok, I'll try that. Thank you!

Reply:
As discussed, we're just going to delete these includes in future